packages/client/hmi-client/src/model-representation/service.ts

```diff
@@ -211,6 +211,16 @@ export function getStates(model: Model): (State & RegNetVertex)[] {
 	}
 }
 
+export function getTransitions(model: Model): Transition[] {
+	const modelType = getModelType(model);
+	switch (modelType) {
```

Comment:
No need for the switch/case blocks, we are only doing Petrinet.

Reply:
Yes I wrote this with the same form as the other getters in this service file and realized we are only supporting petrinet and the other getters are out of date. I will update this